cmd/node/main.go

```diff
@@ -1115,9 +1138,48 @@ func startNode(ctx *cli.Context, log logger.Logger, version string) error {
 	err = networkComponents.NetMessenger.Close()
 	log.LogIfError(err)
 
+	handleAppClose(log, sig)
+
 	return nil
 }
 
+func handleAppClose(log logger.Logger, endProcessArgument endProcess.EndProcessArgument) {
+	log.Debug(fmt.Sprintf("Restarting because %s. Details: %s",
```

Comment:
Why not directly: log.Debug("Restarting", "reason", endProcessArgument.Reason, "details", endProcessArgument.Description)

Reply:
fxied